### ISSUE TITLE
GLTF support

### ISSUE THREAD

**Issue by RoadTrain** [Kromster80/wr_toolkit#19]
Hi! Any plans to support more modern model formats like [GLTF](https://en.wikipedia.org/wiki/GlTF) for import-export in MTKit2?
It's an open standard format which is currently supported by modelling software like Blender and 3ds max.

It looks like there's an Object Pascal library for it: https://github.com/BeRo1985/pasgltf 
I'm not sure how good it is though.

**Comment by Kromster80** [comment]
If anything, I'd say FBX is a modern widespread format that could be nice to add

**Comment by RoadTrain** [comment]
Yeah, FBX is kind of an industry standard right now. The only issue is that it's proprietary, so it's a pain in the ass to implement in open-source. I haven't found any implementations in Delphi/Object Pascal.
